ChangeLog.md

```diff
@@ -1,3 +1,27 @@
+## 1.3.0 - February 2019
+#### Az.Accounts
+* Update to latest version of ClientRuntime
+
+#### Az.AnalysisServices
+General availability for Az.AnalysisServices module.
+
+#### Az.Compute
+* AEM extension: Add support for UltraSSD and P60,P70 and P80 disks
+* Update help description for Set-AzVMBootDiagnostics
+* Update help description and example for Update-AzImage
+
+#### Az.RecoveryServices
+General availability for Az.RecoveryServices module.
+
+#### Az.Resources
+* Fix for issue https://github.com/Azure/azure-powershell/issues/8166
```

Comment:
see comments below - we should include a short (one sentence0 summary for each fixed bug.